test/Groups/homomorphisms.jl

```diff
@@ -96,6 +96,82 @@ end
    
 end
 
+@testset "map_word" begin
+   # Create a free group in GAP in syllable words family,
+   # in order to make the tests.
+   GAP.Globals.PushOptions(GAP.GapObj(Dict(:FreeGroupFamilyType => GAP.GapObj("syllable"))))
+   FS = free_group(2)   # syllable representation
+   GAP.Globals.PopOptions()
+   FL = free_group(2)   # letter representation
+   @test GAP.Globals.IsSyllableWordsFamily(
+           GAP.Globals.ElementsFamily(GAP.Globals.FamilyObj(FS.X)))
+   @test GAP.Globals.IsLetterWordsFamily(
+           GAP.Globals.ElementsFamily(GAP.Globals.FamilyObj(FL.X)))
+   for F in [FL, FS]
+     F1 = gen(F, 1)
+     F2 = gen(F, 2)
```

Comment:
Fun fact: you can also do this:
```suggestion
     F1, F2 = gens(F)
```